toolchain/sem_ir/formatter.cpp

```diff
@@ -1203,6 +1203,11 @@ class Formatter {
   auto FormatArg(InstId id) -> void { FormatInstName(id); }
 
   auto FormatArg(InstBlockId id) -> void {
+    if (!id.is_valid()) {
+      out_ << "invalid";
```

Comment:
Minor, but I think printing `id` here would result in `<invalid>`, which maybe is worth the `<>` bracketing / standardized print?

```suggestion
      out_ << id;
```